pyproject.toml

```
@@ -1,11 +1,98 @@
 [build-system]
-requires = ["setuptools>=61",
+requires = ["setuptools>=64",
             "wheel",
-            "setuptools_scm!=7.0.0,!=7.0.1,!=7.0.2",
+            "setuptools_scm>=8.0",
             "oldest-supported-numpy",
             "cython>=3.0"]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "pyuvdata"
+authors = [
+    {name = "Bryna Hazelton", email = "brynah@phys.washington.edu"},
+    {name = "Adam Beardsley", email = "adam.beardsley@winona.edu"},
+    {name = "Daniel Jacobs", email = "wheresmytab@gmail.com"},
+    {name = "Garrett Keating", email = "garrett.keating@cfa.harvard.edu"},
+    {name = "Nicholas Kern", email = "nkern@mit.edu"},
+    {name = "Matthew Kolopanis", email = "mjkolopa@asu.edu"},
+    {name = "Adam Lanman", email = "alanman@mit.edu"},
+    {name = "Paul LaPlante", email = "paul.laplante@unlv.edu"},
```

```suggestion
    {name = "Paul La Plante", email = "paul.laplante@unlv.edu"},
```
Thanks @bhazelton! Just a small change on how I prefer to spell my name.

---

Oops, sorry! I know better than that. Thanks!